docs/cow-protocol/tutorials/widget/widget.md

```diff
@@ -77,7 +78,31 @@ The fee cannot exceed 1% (100 bps).
 
 :::
 
-The recipient is the address to which the fee will be sent.
+### Recipient
+
+The recipient is the address to which the fee will be sent.  
+
+Make sure that the fee recipient address exists on the respective network defined in the parameters.
+For example, if you use a Safe as a recipient and the Safe was created on Ethereum mainnet, then for Gnosis Chain you must create **another** Safe and set it as a fee recipient.
+
+As a fee recipient, you can specify either string or a key-value pair in the format `chainId: recipientAddress`:
+
+```typescript
+import type {SupportedChainId} from '@cowprotocol/cow-sdk'
+import type {CowSwapWidgetParams} from '@cowprotocol/widget-lib'
+
+const params: CowSwapWidgetParams = {
+  partnerFee: {
+      bps: 50, // 0.5%
+      recipient: {
+          [SupportedChainId.MAINNET]: '0x0000000000000000000000000000000000000001',
+          [SupportedChainId.GNOSIS_CHAIN]: '0x0000000000000000000000000000000000000002',
+          [SupportedChainId.SEPOLIA]: '0x0000000000000000000000000000000000000003',
```

Comment:
While the strong typing with Enums is good for us internally, I wonder whether here (in the example and the actual usage) we could show the usage without cow-sdk.
Not sure whether it's bundled in the widget-lib package already, but I rather not require people to add cow-sdk just for this.

Meaning, what about:

```suggestion
          1: '0x0000000000000000000000000000000000000001',
          100: '0x0000000000000000000000000000000000000002'
```

And also not include Sepolia, because of course there won't be any partner fees paid for it =P